Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM python:3.6
```

Comment:
Should be 3.4.2 to keep inline with Survey Runner? Or is there a reason we need 3.6?